deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/worker/DeployJanitor.java

```diff
@@ -60,13 +60,13 @@ void processDeploys() throws Exception {
                     LOG.debug("Successfully to get the lock: %s", deployLockName);
                     try {
                         deployDAO.deleteUnusedDeploys(envId, timeThreshold, numToDelete);
-                        LOG.info("Successfully removed deploys: %s before %d milliseconds has %d.",
-                            envId, timeThreshold, numToDelete);
+                        LOG.info(String.format("Successfully removed deploys: %s before %d milliseconds has %d.",
+                            envId, timeThreshold, numToDelete));
                     } catch (Exception e) {
                         LOG.error("Failed to delete builds from tables.", e);
```

Comment:
this error need to be changed to: Failed to **remove deploys** from tables. Right now this error is the same as the one when "delete builds" fails, which could make debugging difficult.